FOUNDATIONS.md

```diff
@@ -0,0 +1,160 @@
+# IPLD Foundational Principles
+
+This document outlines parts of IPLD that should and should not be changed to
+ensure the success of future improvements (especially type systems).
+
+**Definitions**
+
+* Block: A block is a chunk of an IPLD DAG, encoded in a format. Blocks have CIDs.
+* Fragment: A piece of an IPLD DAG. Blocks contain fragments.
```

Comment:
What distinguishes a “fragment” from a “node?”

Reply:
Don't worry about that one ;) https://github.com/ipld/specs/pull/152